### PR TITLE
Add and adjust global page title line height

### PR DESCRIPTION
**Pull request by LittleBigThing**
The line height of entry and page titles is reduced from 1.3 to 1.1 for all screens. A take to solve #193.

I am not sure if this is the right approach, so please comment.

### CODE REVIEW

File: assets/css/ie-editor.css [hunk 1]
```diff
@@ -1606,7 +1606,13 @@ pre.wp-block-verse {
 	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
 	font-weight: 300;
 	font-size: 4rem;
-	line-height: 1.3;
+	line-height: 4rem;
```
Comment (melchoyce) [inline assets/css/ie-editor.css]: Is this meant to be rems, instead of unitless?

File: assets/css/ie-editor.css [hunk 2]
```diff
@@ -1606,7 +1606,13 @@ pre.wp-block-verse {
...
+}
+
+@media only screen and (min-width: 652px){
+	.wp-block.editor-post-title__block .editor-post-title__input{
+	line-height: 6rem;
```
Comment (melchoyce) [inline assets/css/ie-editor.css]: Same here ^

File: assets/css/ie-editor.css (hunk 1 above)
Comment (pattonwebz) [reply assets/css/ie-editor.css]: Should be unitless I think

**Comment by LittleBigThing** [comment]
Hm... is that something I added? 🙃 I didn't mean too. I'll check.

**Comment by melchoyce** [comment]
This may have also come in from somewhere else: https://github.com/WordPress/twentytwentyone/pull/212/files#diff-cec66e3dceab2b8961ebb84d2663010bR1749

File: assets/sass/05-blocks/utilities/_editor.scss [hunk 3]
```diff
@@ -15,7 +15,7 @@
 		font-family: var(--heading--font-family);
 		font-weight: var(--heading--font-weight-page-title);
 		font-size: var(--global--font-size-page-title);
-		line-height: var(--heading--line-height);
+		line-height: var(--heading--font-size-h1);
```
Comment (carolinan) [inline assets/sass/05-blocks/utilities/_editor.scss]: I think it is because the line height is set to a font size, not the line height.

**Comment by carolinan** [comment]
The change looks correct, but you need to build the css files again.

**Pull request merged**